### PR TITLE
DMP-3978: Return Batch Size in GET /automated-tasks/{task_id}

### PR DESCRIPTION
**Pull request by sabahirfan**
### JIRA link (if applicable) ###
DMP-3978

### Change description ###
Return Batch Size in GET /automated-tasks/{task_id}

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/openapi/tasks.yaml [hunk 1]
```diff
@@ -132,6 +132,8 @@ components:
           $ref: '#/components/schemas/CronExpression'
         is_active:
           $ref: '#/components/schemas/IsActive'
+        batchSize:
```
Comment (chrisbellingham-hmcts) [inline src/main/resources/openapi/tasks.yaml:135]: Note the `AutomatedTaskSummary` object is also used by the `GET /admin/automated-tasks` endpoint which returns all automated tasks, whereas DMP-3915 only specifies changing the `GET /admin/automated-tasks/{id}` endpoint, so this change will have broader consequences than what the ticket intends.

Suggest this be moved into the `DetatailedAutomatedTask` object instead, which is only used by the get by id endpoint?

Also the naming convention for API properties is snake_case (see https://hmcts.github.io/restful-api-standards/#118), so the attribute name on L135 should be `batch_size`.

File: src/main/resources/openapi/tasks.yaml [hunk 2]
```diff
@@ -132,6 +132,8 @@ components:
...
+          $ref: '#/components/schemas/batchSize'
```
Comment (chrisbellingham-hmcts) [inline src/main/resources/openapi/tasks.yaml:136]: Please capitalise this to BatchSize for consistency with the other component names.

**Review by chrisbellingham-hmcts — COMMENTED** [review]
Is there any way we can enhance `AdminGetAutomatedTasksByIdTest` to verify that the full (JSON serialised) API response is as expected?

File: src/main/resources/openapi/tasks.yaml [hunk 3]
```diff
@@ -164,6 +166,8 @@ components:
       maxLength: 256
     IsActive:
       type: boolean
+    batchSize:
```
[the remark this answers is not in the document]
Comment (sabahirfan) [reply src/main/resources/openapi/tasks.yaml]: done

File: src/main/resources/openapi/tasks.yaml (hunk 2 above)
Comment (sabahirfan) [reply src/main/resources/openapi/tasks.yaml]: done

**Review by chrisbellingham-hmcts — APPROVED** [review]

**Review by jackmaloney — APPROVED** [review]

**Pull request merged**